regression/esbmc-solidity/Makefile

```diff
@@ -0,0 +1,19 @@
+default: tests.log
```

Comment:
You don't need this file. Add the directory here: https://github.com/esbmc/esbmc/blob/master/regression/CMakeLists.txt

Reply:
Done.